db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/subquery/aggregate/AggregateSubqueryPredicate.java

```diff
@@ -88,6 +118,10 @@ private boolean doWeHandle(SubqueryNode subqueryNode) throws StandardException {
                 (subquerySelectNode.getGroupByList() == null || subquerySelectNode.getGroupByList().isEmpty()))
             return false;
 
+        /* the aggregate must not be value-producing */
```

Comment:
Can you add here a short explanation or an example of what we want to avoid?

Reply:
Sure